// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

syntax = "proto2";
package hit.protobuf;

import "ciphertext.proto";

message Ciphertext_Vector {
	repeated Ciphertext cts = 1; // A list of Ciphertext.
}
